ml-algorithms/build.gradle

```diff
@@ -24,7 +24,7 @@ dependencies {
     implementation files('lib/randomcutforest-parkservices-2.0.1.jar')
     implementation files('lib/randomcutforest-core-2.0.1.jar')
     testImplementation group: 'junit', name: 'junit', version: '4.12'
-    testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.9.0'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '4.4.0'
     testImplementation group: 'org.mockito', name: 'mockito-inline', version: '3.9.0'
```

Comment:
I see the we use latest version of `mockito-core`. Why not use the latest version of `mockito-inline`: `4.4.0`?

Reply:
good catch. this is a left over. should use 4.4 for consistency. Updated in the revision.